docs/pages/router/reference/testing.mdx

```diff
@@ -173,3 +180,67 @@ expect(screen).toHaveRouterState({
 ```
 
 </PaddedAPIBox>
+
+## Examples
+
+For a simple component, where you're not explicitly asserting against or invoking any `expo-router` behavior, you may simply need to render with `renderRouter` instead of `render`. If you're not testing route `pathname`s, then you can simply do this by rendering your component as the `index`:
+
+```tsx app.test.ts
+renderRouter({
+  index: jest.fn(() => <MyComponent />),
+});
+```
+
+If you wish to assert against navigation actions, you may need to mock the `router` function, as well as any other functions you wish to assert against:
+
+```tsx app.test.ts
+jest.mock('expo-router', () => ({
+  ...jest.requireActual('expo-router'),
+ useLocalSearchParams: jest.fn().mockImplementation(() => ({ // or use `mockReturnValue`
+    someParam: foo-param,
+  router: {
+    navigate: jest.fn(),
+  },
+}));
+
+...
+
+expect(router.navigate).toHaveBeenCalledWith('/profile');
+```
```

Comment:
I'm not sure I agree with this section. If someone is testing a screen and not the behaviour, then they can just import the screen directly into `render()` (or have the screen just render a single component and test that component).

If you are testing that the navigation actions were fired, then you should allow navigation to occur and then assert against the new pathname of the screen.

By mocking `expo-router` its very easy to create a false-positive where you think events are firing, but these events may actually be invalid and navigation never occurs.